pom.xml

```diff
@@ -167,7 +169,7 @@
           <target>
             <artifact>
               <groupId>de.cau.cs.kieler.klighd</groupId>
-              <artifactId>de.cau.cs.kieler.klighd.targetplatform</artifactId>
+              <artifactId>de.cau.cs.kieler.klighd.targetplatform-elkNightly</artifactId>
```

Comment:
Remember to remove this change before merging.

Reply:
Can be resolved once ELK 0.7.0 is released.